nss-day-cohort-60/national-parks-api
# Jd 2 fetch parameters refactor

Moved 2 parameter blog fetch function into blog_views.py for structural consistency, added blog search function to blog_views and method_mapper.

#TESTING:
1. pull and switch to JD-2-fetch-parameters-refactor branch
2. Confirm 2 param fetch still functions in postman with http://localhost:8088/blogs?park_id=8&key_word=tree 
3. Confirm search works in postman with http://localhost:8088/blogs?key_word=tree and http://localhost:8088/blogs?key_word=tre
      All should return status 200, key_word=tre should return 6 blog objects, key_word=tree should return 3
